### ISSUE TITLE
valid-jsdoc problem with class properties

### ISSUE THREAD

**Issue by webOS101** [babel/eslint-plugin-babel#111]
**Tell us about your environment**

* **ESLint Version:** 3.9.1
* **Node Version:** 6.8.0
* **npm Version:** 3.10.9

**What parser (default, Babel-ESLint, etc.) are you using?**  eslint-plugin-babel@3.3.0

**Please show your full configuration:**

```
{
	env: {
		'es6': true, // sets the "ecmaVersion" parser option to 6
	},
	parser: 'babel-eslint',
	plugins: [
		'babel'
	],
	rules: {
		'valid-jsdoc': 1
	}
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
/**
 * @class Whoops
 */
class Whoops {
	test = (wow) => {
		return wow;
	}
}

export default Whoops;
```

**What did you expect to happen?**  No warning, `test` does not have a jsdoc tag

**What actually happened? Please include the actual, raw output from ESLint.**
`  1:1  warning  Missing JSDoc for parameter 'wow'  valid-jsdoc`

If the method is changed to `test(wow) { }` then no warning is shown.

eslint reference: https://github.com/eslint/eslint/issues/7555

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
